python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/ethernet_interfaces.schema.yml

```diff
@@ -880,6 +883,43 @@ keys:
                       max: 65535
                       convert_types:
                         - str
+            aaa:
+              type: dict
+              keys:
+                unresponsive:
+                  type: dict
+                  description: Configure AAA timeout options.
+                  keys:
+                    eap_response:
+                      type: str
+                      description: EAP response to send. EOS default is `success`.
+                      valid_values:
+                        - "success"
+                        - "disabled"
+                    action:
+                      type: dict
+                      keys:
+                        apply_cached_results:
+                          type: bool
+                          $ref: "eos_cli_config_gen#/keys/dot1x/keys/aaa/keys/unresponsive/keys/action/keys/apply_cached_results"
```

Comment:
instead of adding reference for each key, you can add a ref in `action` and define only `traffic_allow_access_list ` key here

Reply:
Fixed.